bin/cake.php

```diff
@@ -16,17 +16,8 @@
  * @license       https://opensource.org/licenses/mit-license.php MIT License
  */
 
-$minVersion = '5.6.0';
-if (file_exists('composer.json')) {
-    $composer = json_decode(file_get_contents('composer.json'));
-    if (isset($composer->require->php)) {
-        $minVersion = preg_replace('/([^0-9\.])/', '', $composer->require->php);
-    }
-}
-if (version_compare(phpversion(), $minVersion, '<')) {
-    fwrite(STDERR, sprintf("Minimum PHP version: %s. You are using: %s.\n", $minVersion, phpversion()));
```

Comment:
I can reintroduce fwrite to STDERR if you think this is better than just trigger_error() in requirements.php